src/store/teleSlice.js

```diff
@@ -0,0 +1,33 @@
+import { createSlice } from "@reduxjs/toolkit"
+
+const initialState = {
+    orientW: null,
+    orientX: null,
+    orientY: null,
+    orientZ: null,
+    posX: null,
+    posY: null,
+    posZ: null,
+    timestamp: null
+}
+
+const teleSlice = createSlice({
```

Comment:
`tele` is an awkward prefix. Can we call this `telemetrySlice`? It's not *that* long and it's more explicit.